package.json

```diff
@@ -12,6 +12,7 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
+    "classnames": "^2.5.1",
```

Comment:
Classname should be removed from here too.

Reply:
Done. Thanks